src/label/labelStyle.ts

```diff
@@ -275,6 +276,7 @@ function setLabelStyle<TLabelDataIndex>(
         textContent.ignore = !showNormal;
         // Always create new style.
         textContent.useStyle(normalStyle);
+        textContent.z = opt.z ?? textContent.z;
```

Comment:
We should be careful about using `??` or `?.` because they will be transpiled to much more complex code and increase the final bundle size. In this case:
```ts
opt.z != null && (textContent.z = opt.z);
```
is clearer and won't be transpiled.